apps/test/unit/templates/progress/ProgressLevelSetTest.js

```diff
@@ -30,10 +30,10 @@ describe('ProgressLevelSet', function () {
 
     assert.equal(wrapper.find('ProgressPill').length, 1);
     assert.equal(wrapper.find('ProgressBubbleSet').length, 1);
-    assert.equal(wrapper.find('ProgressPill').props().text, 'LEVEL 1-3');
+    assert.equal(wrapper.find('ProgressPill').props().text, '1-3');
   });
 
-  it('renders a pill that says UNPLUGGED when first level is unplugged', () => {
+  it('renders a pill with no text when first level is unplugged', () => {
```

Comment:
Should there be a matching test "renders a pill with no text when _last_ level is unplugged?"